redbot/cogs/audio/converters.py

```diff
@@ -25,50 +27,42 @@
     "get_playlist_converter",
 ]
 
-if TYPE_CHECKING:
-    _bot: Red
-    _config: Config
-else:
-    _bot = None
-    _config = None
-
-_SCOPE_HELP = """
+_SCOPE_HELP: Final[
+    str
+] = """
 Scope must be a valid version of one of the following:
 ​ ​ ​ ​ Global
 ​ ​ ​ ​ Guild
 ​ ​ ​ ​ User
 """
```

Comment:
THIS IS PART 1/3 THAT NEED TO BE APPLIED TOGETHER WITH THIS.

```suggestion
T_ = _
_ = lambda s: s

_SCOPE_HELP: Final[str] = _(
    """
Scope must be a valid version of one of the following:
​ ​ ​ ​ Global
​ ​ ​ ​ Guild
​ ​ ​ ​ User
"""
)
```